modules/mssql/mssql.go

```diff
@@ -0,0 +1,89 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const defaultImage = "mcr.microsoft.com/mssql/server:2022-latest"
```

Comment:
NIT: please combine the constants into the same block as in https://github.com/testcontainers/testcontainers-go/pull/1969/files#diff-c0e7ade19e017ade3c713ff8bdcc34da96c6f5f6543c57db55d643093815c1ccR14-R18